format_spec.md

```diff
@@ -0,0 +1,339 @@
+
+##### `caption` (optional)
+
+An optional string which can be displayed as a caption for the current Gallery Item. Any form of rich text such as HTML CANNOT be used. `caption` CANNOT be present when `rel` is set to `icon`.
+
+##### `sha256` (optional)
+
+An optional SHA256 checksum for the media file. This field CANNOT be present if the `embed` field is present.
+
+### `authors` (optional)
+
+This field MUST be stored as an array. It contains objects storing the information about each author. Those objects MUST be EITHER a Team or an individual Author.
+
+**Team objects consist of the following fields:**
+
+##### `team`
+
+Teams MUST have a `team` field that is set to true. A `team` field that is not set to `true` is invalid.
+
+##### `id`
+
+A unique identifier to this team, allowing you to search by Team. There is no required format for how this ID should be generated, however it MUST be unique to the Team
+
+##### `name`
+
+The human readable name of the team. Spaces are allowed.
+
+##### `uri` (optional)
+
+An optional field that can link to a webpage relating to the Team.
+
+##### `members`
+
+An array containing Author objects, who are the members of the team.
+
+**Author objects consist of the following fields:**
+
+##### `id`
+
+A unique identifier to this author. The same requirements for Team IDs should be followed here. This ID MUST be unique to the author.
+
+##### `name`
+
+The human readable name of the author. Spaces are allowed.
+
+##### `uri` (optional)
+
+An optional field that specifies a website relating to the author. It may be the author's personal website, the author's profile on a certain host, or any valid URL.
+
+---
+
+### `categories` (optional)
+
+This field MUST be stored as an array. It contains Category Objects. Those objects MUST be made up of the following fields:
+
+##### `name`
+
+The human readable name of the category. Examples include "Magic", "Technology", and "World Gen". It is NOT RECOMMENDED to use the name of a modloader as a category.
+
+##### `uri` (optional)
+
+An optional link to the category on a host, where users could see Projects within that category.
+
+##### `icon` (optional)
+
+This is an object that contains the following fields:
+
+##### `sha256` (optional)
+
+An optional SHA256 checksum of the icon specified in `uri`.
+
+##### `uri`
+
+The URI of this icon. This field is required to be present if the `icon` object is present.
+
+---
+
+### `license` (optional)
+
+This is an object that contains information about the license that a project is under. It contains the following fields:
+
+##### `spdx`
+
+The SPDX identifier of the license. [You can find a list of SPDX license names and identifiers here](https://spdx.org/licenses/).
+
+##### `uri`
+
+A URL linking to the full license.
+
+##### `modpackUsage` (optional)
+
+A field describing usage of the Project in modpacks. Valid options:
+
+- `alwaysAllow` - This Project can be used in a modpack without asking for permission.
```

Comment:
Does it override the license? If modpackUsage is alwaysAllow but the license says no redistribution, is the mod allowed to be redistributed as part of a modpack?